src/number-formatting/short-format.js

```diff
@@ -20,6 +20,12 @@ import formatNumber from './format';
  * @returns {string} Returns 0 when dates are equal. -1 when date1 less than date2. 1 when date1 greater than date2.
  */
 function shortFormat(num, options) {
+    let minusSign = '';
+    if (num < 0) {
+        minusSign = '-';

```

Comment:
This isn't culturally correct - price formatting uses negativePre and negativePost
number format uses negativePattern

Reply:
I assume we are doing some cultural things so that "," or "." is used correctly?

Reply:
Got it! I have updated the code and added more test cases for to check for `negativePost` scenario.